src/lib/i18n/dictionaries/dictionary.json

```diff
@@ -3534,14 +3534,14 @@
   "description.camel": "A mean looking camel!",
   "description.tomato.bombard": "Home to Cannonball, and is ready to strike anyone who gets in its way",
   "keys": "Keys",
-  "description.butterfly": "Butterfly loves the scent of flowers. 20% chance of +1 Flower",
-  "description.macaw": "Macaw loves picking fruits. +0.1 Fruit",
-  "description.squirrel": "Squirrel likes hanging out in the forest. +0.1 Wood",
-  "description.fairy.circle": "Circle of fairy mushrooms. +0.2 Wild Mushroom",
-  "description.gold.beetle": "Beetle made of gold. +0.1 Gold",
-  "description.iron.beetle": "Beetle made of iron. +0.1 Iron",
-  "description.stone.beetle": "Beetle made of stone. +0.1 Stone",
-  "description.macaw.boost": "+0.1 Fruit",
+  "description.butterfly": "Butterfly loves the scent of flowers.",
+  "description.macaw": "Macaw loves picking fruits.",
+  "description.squirrel": "Squirrel likes hanging out in the forest.",
+  "description.fairy.circle": "Circle of fairy mushrooms.",
+  "description.gold.beetle": "Beetle made of gold.",
+  "description.iron.beetle": "Beetle made of iron.",
+  "description.stone.beetle": "Beetle made of stone.",
+  "description.macaw.camel.onesie.boost": "+0.1 Fruit Patch Yield",
```

Comment:
Even though it's the same term, we normally would separate the terms into separate terms. could you separate them?